jung-visualization/src/main/java/edu/uci/ics/jung/visualization/spatial/SpatialQuadTree.java

```diff
@@ -134,13 +138,15 @@ protected void split() {
         new SpatialQuadTree<>(layoutModel, childLevel, x, y + height, width, height);
     SpatialQuadTree<N> se =
         new SpatialQuadTree<>(layoutModel, childLevel, x + width, y + height, width, height);
-    children =
-        new ImmutableMap.Builder<Quadrant, SpatialQuadTree<N>>()
-            .put(NE, ne)
-            .put(NW, nw)
-            .put(SW, sw)
-            .put(SE, se)
-            .build();
+    synchronized (lock) {
+      children =
+          new ImmutableMap.Builder<Quadrant, SpatialQuadTree<N>>()
```

Comment:
Note that you can do this instead:

    children = ImmutableMap.of(NE, ne, NW, nw, SW, sw, SE, se);

I think that you can create an ImmutableMap of size up to 5 using of() in this way.

Reply:
thanks! I was looking for that.